.travis.yml

```diff
@@ -28,17 +28,17 @@ addons:
       - llvm-toolchain-precise
 
 env:
-  - COMPILER=clang++-3.6 CMAKE_GENERATOR="Ninja"
-  - COMPILER=clang++-3.6 CMAKE_GENERATOR="Unix Makefiles"
-
-  - COMPILER=clang++-3.7 CMAKE_GENERATOR="Unix Makefiles"
-  - COMPILER=clang++-3.7 CMAKE_GENERATOR="Ninja"
-
-  - COMPILER=clang++-3.8 CMAKE_GENERATOR="Unix Makefiles"
-  - COMPILER=clang++-3.8 CMAKE_GENERATOR="Ninja"
-
-  - COMPILER=gcc CMAKE_GENERATOR="Ninja"
-  - COMPILER=gcc CMAKE_GENERATOR="Unix Makefiles"
+  # Test all generators and Clang + GCC
+  - TEST=true COMPILER=clang++-3.8 CMAKE_GENERATOR="Ninja"
+  - TEST=true COMPILER=clang++-3.8 CMAKE_GENERATOR="Unix Makefiles"
+  - TEST=true COMPILER=gcc         CMAKE_GENERATOR="Ninja"
+  - TEST=true COMPILER=gcc         CMAKE_GENERATOR="Unix Makefiles"
+
+  # Benchmarks
+  - BENCHMARK=true COMPILER=clang++-3.6 CMAKE_GENERATOR="Unix Makefiles"
+  - BENCHMARK=true COMPILER=clang++-3.7 CMAKE_GENERATOR="Unix Makefiles"
+  - BENCHMARK=true COMPILER=clang++-3.8 CMAKE_GENERATOR="Unix Makefiles"
+  - BENCHMARK=true COMPILER=gcc         CMAKE_GENERATOR="Unix Makefiles"
```

Comment:
My reasoning was that as long as we test on different generators and different compilers (GCC/Clang), the version of the compiler should not matter much for testing Metabench. However, it is interesting to see how a specific version of a compiler behaves in terms of compile-time, because people will be interested in seeing how the exact version they use behaves, not just the latest one.

I can add testing on all Clang versions if you think it is useful, but I would definitely benchmark on many versions.